src/GeneExpressionTSnePlot.js

```diff
@@ -67,7 +67,7 @@ const _colourizeExpressionLevel = (gradientColours, highlightSeries) => {
               return {
                 ...point,
                 expressionLevel: Math.round10(point.expressionLevel, -2),
-                colorv: Math.round10(point.expressionLevel, -2)
+                color_value: Math.round10(point.expressionLevel, -2)
```

Comment:
Please follow conventions when naming variables. We use `camelCase` for naming variables, rather than `snake_case`.